tests/App/config/services.yml

```diff
@@ -1,9 +1,16 @@
 services:
-    # Under real conditions, this service is registered by "doctrine/doctrine-bundle"
-    database_connection:
-        class: Sonata\AdminBundle\Tests\Fixtures\DoctrineBundle\Connection
+    _defaults:
+        autowire: true
+        autoconfigure: true
+        public: false
```

Comment:
this can be removed since we require at least Symfony 4, right? Its the default then.